nixos/modules/security/pam.nix

```
@@ -489,6 +489,20 @@ let
         package = mkPackageOption pkgs.plasma5Packages "kwallet-pam" {
           pkgsText = "pkgs.plasma5Packages";
         };
+
+        ttyUnlock = mkEnableOption {
+          default = false;
+          type = types.bool;
+          description = ''
+            The auto-lock only works when you log-in from a Display Manager.
+            For many Wayland-based window managers (Sway, Hyprland, etc),
+            logging in from a TTY is the only officially supported method
+            (even though logging in from a Display Manger should work). If you
+            intend to log-in from a TTY, it is recommended that you enable
+            this option **and** ensure that `plasma-kwallet-pam.service` is
+            triggered by `graphical-session.target`.
+          '';
+        };
```

Actually
```suggestion
        ttyUnlock = mkEnableOption null // {
          description = ''
            The auto-lock only works when you log-in from a Display Manager.
            For many Wayland-based window managers (Sway, Hyprland, etc),
            logging in from a TTY is the only officially supported method
            (even though logging in from a Display Manger should work). If you
            intend to log-in from a TTY, it is recommended that you enable
            this option **and** ensure that `plasma-kwallet-pam.service` is
            triggered by `graphical-session.target`.
          '';
        };
```

Here's the documentation: https://github.com/hsjobeki/nixpkgs/blob/migrate-doc-comments/lib/options.nix#L137:C3

the null is there to prevent automatic description like `Whether to enable ....`

---

missed that, sorry, force pushed again